src/sidebar/components/test/tag-editor-test.js

```diff
@@ -307,4 +307,37 @@ describe('TagEditor', function() {
       assert.isTrue(fakeOnEditTags.calledWith({ tags: ['tag2'] }));
     });
   });
+
+  describe('filter on text input based on user agent', () => {
+    const originalUA = navigator.userAgent;
+
+    afterEach(function() {
+      navigator.__defineGetter__('userAgent', () => {
```

Comment:
I could not figure out a better way to mock this :\

Reply:
You can use `sinon.stub`:

```js
// Stub property.
let stub = sinon.stub(navigator, 'userAgent');

// Define getter for stub that returns a fixed string.
stub.get(() => 'Chrome/1.0')

// Remove the stub
stub.restore()
```

Alternatively you could add a test seam to the component - add a `userAgent` prop which is default-initialized to `navigator.userAgent`.